docs/v2-system-addon/1.GETTING_STARTED.md

```diff
@@ -0,0 +1,74 @@
+# Activity Stream (System add-on)
+
+## Where to ask questions
+
+- Most of the core dev team can be found on the `#activity-stream` mozilla irc channel.
+  You can also direct message the core team (`dmose`, `emtwo`, `jkerim`, `k8hudson`, `Mardak`, `nanj`, `r1cky`, `ursula`)
+  or our manager (`tspurway`)
+- Our Mozilla slack channel is #activitystream
+- File issues/questions on Github: https://github.com/mozilla/activity-stream/issues
+
+## Testing in Firefox Nightly
+
+If you just want to see the current version of Activity Stream in Firefox, you can
+install [Firefox Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/)
```

Comment:
Might as well link to https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly with the anchor